proxy/logging/LogObject.h

```diff
@@ -97,7 +97,6 @@ class LogObject : public RefCountObj
             const char *header, Log::RollingEnabledValues rolling_enabled, int flush_threads, int rolling_interval_sec = 0,
             int rolling_offset_hr = 0, int rolling_size_mb = 0, bool auto_created = false, int rolling_max_count = 0,
             int rolling_min_count = 0, bool reopen_after_rolling = false, int pipe_buffer_size = 0);
-  LogObject(LogObject &);
```

Comment:
We should probably delete this and the copy assignment constructors to make sure they don't accidentally get used in the future.

> LogObject(LogObject const&) = delete;
> LogObject& operator=(LogObject const&) = delete;

Reply:
These are already present:  https://github.com/apache/trafficserver/blob/e6291181b4eb6bc98b57fa1cd86ec07fd2d59928/proxy/logging/LogObject.h#L300
Although they are private, which would make for a confusing error message I think.  So the defined copy constructor is especially krufty, it only applies to non-const source objects.

Reply:
Ah, ok. Thanks!